src/OpenLoco/src/Map/Track/TrackData.cpp

```diff
@@ -355,7 +355,7 @@ namespace OpenLoco::World::TrackData
             .compatibleFlags = TrackTraitFlags::none,
             .curveSpeedFraction = 0xFFFF,
             .unkWeighting = 32,
-            .sparkDirection = true },
+            .sparkDirection = true }, // straight
```

Comment:
I hate to nitpick, but I think it would be more helpful if these were on their own line, before the `TrackMiscData{` or `RoadMiscData{`.